packages/cryptoplease/lib/app/screens/authenticated/profile/components/danger_section.dart

```diff
@@ -13,6 +13,7 @@ class DangerSection extends StatelessWidget {
           sl<ISKPRepository>().clear();
           sl<ODPRepository>().clear();
           sl<OSKPRepository>().clear();
+          sl<IntercomService>().logout();
```

Comment:
What if the user will log out and then log in again? Intercom won't be re-initialized.

Reply:
You're right, I've updated pr to have a separate login call in onboarding after creation/restore of accounts